var/spack/repos/builtin/packages/qmcpack/package.py

```diff
@@ -255,6 +314,23 @@ def install(self, spec, prefix):
             # install binaries
             install_tree('bin', prefix.bin)
 
+    # QMCPACK 3.6.0 install directory structure changed, thus there
+    # thus are two version of the setup_environment method
+    @when('@:3.5.0')
+    def setup_environment(self, spack_env, run_env):
+        """Set-up runtime environment for QMCPACK.
+        Set PYTHONPATH for basic analysis scripts and for Nexus."""
+        run_env.prepend_path('PYTHONPATH', join_path(self.prefix, 'nexus'))
```

Comment:
You can actually use `self.prefix.nexus` here, no `join_path` necessary.

Reply:
I can address this.